src/app/features/project/project.model.ts

```diff
@@ -79,12 +79,20 @@ export type WorklogColTypes =
   | 'ESTIMATE_CLOCK'
   ;
 
+export enum WorklogGrouping {
+  DATE = 'DATE',
```

Comment:
+1 for enum :)

Reply:
Angular uses Typescript for a reason, right? :P